spek-ide-plugin/intellij-common/src/main/kotlin/org/spekframework/intellij/SpekRunLineMarkerContributor.kt

```diff
@@ -0,0 +1,21 @@
+package org.spekframework.intellij
+
+import com.intellij.execution.lineMarker.ExecutorAction
+import com.intellij.execution.lineMarker.RunLineMarkerContributor
+import com.intellij.icons.AllIcons
+import com.intellij.psi.PsiElement
+import com.intellij.util.Function
+
+class SpekRunLineMarkerContributor: RunLineMarkerContributor() {
+
+    override fun getInfo(element: PsiElement): Info? {
+        val path = extractPath(element)
+        return path?.let {
+            Info(
+                AllIcons.RunConfigurations.TestState.Run,
+                Function<PsiElement, String> { "Run ${path.name}"},
```

Comment:
If you use `it` here (which is non-nullable in this scope) you can avoid the `path` variable.

Reply:
ack

Reply:
A reminder in case you forgot this.

Reply:
I still need to use path, since `it` in `Function<PsiElement, String> { "Run ${path.name}"}` is a `PsiElement` not a `Path`.